src/backend/distributed/deparser/ruleutils_11.c

```diff
@@ -4812,6 +4812,8 @@ get_rule_expr(Node *node, deparse_context *context,
 	CHECK_FOR_INTERRUPTS();
 	check_stack_depth();
 
+	node = RelabelTypeMutator(node);
```

Comment:
get_rule_expr gets called a lot, so this walker will likely cause excessive CPU. 

Is there a way to integrate the logic from RelabelTypeMutator into the switch statement below?